src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter((item) => {
+    if (query === '') {
+      return item;
+    }
+
+    if (item.title.toLowerCase().includes(query.toLowerCase())
+    || item.description.toLowerCase().includes(query.toLowerCase())) {
+      return item;
+    }
```

Comment:
Just return it. It will turn into true or fall, what is needed for the filter method. And optional move query.toLowerCase() to a separate const
```suggestion
    return  (
     item.title.toLowerCase().includes(query.toLowerCase())
      || item.description.toLowerCase().includes(query.toLowerCase())
    )
```